Resources/Catalog/FileCatalog.py

```diff
@@ -1,13 +1,62 @@
 """ File catalog class. This is a simple dispatcher for the file catalog plug-ins.
     It ensures that all operations are performed on the desired catalogs.
+
```

Comment:
I am sorry but I really don't like the changes here. The test of the arguments and so on, fine. But putting back the metadata method into the same list (as they were before btw), and then making an "exclusion list", is just recoding the same thing in a different way, with no obvious benefit at least to my eyes (but I might be wrong !). We had a few hour long discussion on separating them. You did not succeed, fine, no problem, we could look together and see if we find an elegant way. But then I would leave the logic untouched here, because that is yet one more thing to test, certify, debug, while it is supposed to do the exact same. if you wanna have a quick v6r15, okay, but then let's do things properly, and move this feature all together to v6r16, where we can have a clean separation of the objects